### PR TITLE
Various docs changes.

### PR DESCRIPTION
**Pull request by costdev**

### CODE REVIEW

File: src/wp-admin/includes/class-wp-upgrader.php [hunk 1]
```diff
@@ -94,12 +94,13 @@ public function __construct() {
...
 	 *
-	 * @return string|WP_Error
+	 * @return string|WP_Error The path to the source location, or WP_Error on failure.
 	 */
 	public function create_backup( $source, $remote_source, $upgrader, $hook_extra ) {
 		$this->options = ( new WP_Plugin_Theme_Upgrader() )->set_callback_options( $hook_extra );
 
-		// Early exit if $hook_extra is empty or if this is an installation and not update.
+		// Early exit if $hook_extra is empty,
+		// or if this is an installation and not update.
```
Comment (afragen) [inline src/wp-admin/includes/class-wp-upgrader.php]: Did you really want this reverted?

Comment (costdev) [reply src/wp-admin/includes/class-wp-upgrader.php]: Fixed now 😆

File: src/wp-admin/includes/class-wp-upgrader.php [hunk 2]
```diff
@@ -124,19 +125,22 @@ public function create_backup( $source, $remote_source, $upgrader, $hook_extra )
...
 	 * @param bool|WP_Error $result     Result from `WP_Upgrader::install_package()`.
 	 * @param array         $hook_extra Array of data for plugin/theme being updated.
 	 *
-	 * @return bool|WP_Error
+	 * @return bool|WP_Error The result from `WP_Upgrader::install_package()`, or WP_Error on failure.
 	 */
 	public function restore_backup( $result, $hook_extra ) {
-		// Early exit if $hook_extra is empty or if this is an installation and not update.
+		// Early exit if $hook_extra is empty,
+		// or if this is an installation and not update.
```
Comment (afragen) [inline src/wp-admin/includes/class-wp-upgrader.php]: I thought you didn't like this?

Comment (costdev) [reply src/wp-admin/includes/class-wp-upgrader.php]: Fixed now! 😆

**Pull request merged**